packages/frontend-2/components/viewer/resources/ActiveVersionCard.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <div class="bg-primary-muted flex gap-2 py-2 px-4">
+    <EyeIcon v-tippy="'Currently viewing'" class="h-4 w-4 text-primary mt-0_5" />
+    <div class="flex flex-col overflow-hidden text-xs select-none">
+      <div class="inline-block rounded-full font-bold">
+        {{ version.sourceApplication }}
+      </div>
+      <div class="truncate text-foreground-2">
+        {{ version.message || 'no message' }}
+      </div>
+      <div class="italic text-foreground-2">
+        {{ timeAgo(version.createdAt) }}
+      </div>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import dayjs from 'dayjs'
+import relativeTime from 'dayjs/plugin/relativeTime'
+import { EyeIcon } from '@heroicons/vue/24/solid'
+
+interface Version {
+  sourceApplication?: string | null
+  message?: string | null
+  createdAt: string
+}
+
+defineProps({
+  version: {
+    type: Object as () => Version,
+    required: true
+  }
+})
+
+dayjs.extend(relativeTime)
+
+const timeAgo = (date: string) => {
```

Comment:
This could be a computed. That would be better from a performance perspective, because a computed is re-calculated only when any of its dependencies change (e.g. props.createdAt), but if you just use a method in your template, it's going to be invoked every single time the component gets re-rendered.